aspnet/mvc/views/forms/sample/final/ViewModels/Person.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace FormsTagHelper.ViewModels
+{
+    public class Person
+    {
+        public Person() { }
+        public List<string> Colors { get; set; }
+        public int Age { get; set; }
```

Comment:
Add blank lines between these properties and remove the explicit default constructor (just noise).